src/FrontEnd/S390/S390Instruction.fs

```diff
@@ -49,13 +63,32 @@ type S390Instruction (addr, numBytes, insInfo) =
   override this.IsIndirectBranch () =
     this.IsBranch () && (not <| this.HasConcJmpTarget ())
 
-  override _.IsCondBranch () = Terminator.futureFeature ()
+  override __.IsCondBranch () =
```

Comment:
Same for this method and the rest. `__` -> `this` or `_` depending on the context.